utils/src/main/java/software/amazon/awssdk/utils/IoUtils.java

```diff
@@ -63,16 +61,25 @@ public static String toUtf8String(InputStream is) throws IOException {
      * Closes the given Closeable quietly.
      * @param is the given closeable
      * @param log logger used to log any failure should the close fail
+     * @deprecated use {@link #closeQuietlyV2(AutoCloseable, Logger)} instead
      */
-    public static void closeQuietly(AutoCloseable is, Logger log) {
+    @Deprecated
+    public static void closeQuietly(AutoCloseable is, org.slf4j.Logger log) {
+        closeQuietlyV2(is, log == null ? DEFAULT_LOG : new Logger(log));
+    }
+
+    /**
+     * Closes the given Closeable quietly.
+     * @param is the given closeable
+     * @param log logger used to log any failure should the close fail
+     */
+    public static void closeQuietlyV2(AutoCloseable is, Logger log) {
```

Comment:
We can't use the same signature because it'll break people who invoke `closeQuietly(closable, null)`